lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -27,7 +27,8 @@ const selectionState = require('../../../../shared/models/selection-state')
 module.exports = function (
   $q,
   $uibModalInstance,
-  WarningModalService
+  WarningModalService,
```

Comment:
Should we remove this? Or are there plans for a warning modal?

Reply:
There are other functions in this controller that use this service.